frontend/components/ActivityItem/ActivityItem.tsx

```diff
@@ -97,7 +97,7 @@ const ActivityItem = ({
   onShowDetails = noop,
   onCancel = noop,
 }: IActivityItemProps) => {
-  const { actor_email } = activity;
+  const { actor_email, actor_api_only } = activity;
```

Comment:
Looks like this `actor_api_only` is unused, you could use it on 146 if you want, or remove from here

Reply:
fixede!